centos_ci.sh

```diff
@@ -6,63 +6,157 @@ set -x
 # Exit on error
 set -e
 
+# GitHub user
+REPO_OWNER="minishift"
+REPO_NAME="minishift-centos-iso"
+
+########################################################
+# Exit with message on failure of last executed command
+# Arguments:
+#   $1 - Exit code of last executed command
+#   $2 - Error message
+########################################################
+function exit_on_failure() {
+  if [[ "$1" != 0 ]]; then
+    echo "$2"
+    exit 1
+  fi
+}
+
 # Source environment variables of the jenkins slave
 # that might interest this worker.
-if [ -e "jenkins-env" ]; then
-  cat jenkins-env \
-    | grep -E "(JENKINS_URL|GIT_BRANCH|GIT_COMMIT|BUILD_NUMBER|ghprbSourceBranch|ghprbActualCommit|BUILD_URL|ghprbPullId|CICO_API_KEY|GITHUB_TOKEN)=" \
-    | sed 's/^/export /g' \
-    > ~/.jenkins-env
-  source ~/.jenkins-env
-fi
+function load_jenkins_vars() {
+  if [ -e "jenkins-env" ]; then
+    cat jenkins-env \
+      | grep -E "(JENKINS_URL|GIT_BRANCH|GIT_COMMIT|BUILD_NUMBER|ghprbSourceBranch|ghprbActualCommit|BUILD_URL|ghprbPullId|CICO_API_KEY|GITHUB_TOKEN|JOB_NAME|RELEASE_VERSION)=" \
+      | sed 's/^/export /g' \
+      > ~/.jenkins-env
+    source ~/.jenkins-env
+  fi
 
-# Enable extra packages
-yum --enablerepo=extras install -y epel-release
+  echo 'CICO: Jenkins ENVs loaded'
+}
 
+function install_required_packages() {
+  # Install EPEL repo
+  yum -y install epel-release
+  # Get all the deps in
+  # We are installing golang from offical repository to make sure our downstream builds works as expected.
+  # CDK side golang always comes from the offical RHEL repository.
```

Comment:
And where is golang package listed here?

Reply:
It is not required as part of this task.